parachains/integration-tests/collectives/1_alliance/1_init_alliance.yml

```diff
@@ -159,13 +181,26 @@ tests:
               pallet: xcmPallet
               call: send
               args: [
-                { v1: { parents: 0, interior: { x1: { parachain: *coll_para_id }}}}, # destination
+                { v3: { parents: 0, interior: { x1: { parachain: *coll_para_id }}}}, # destination
                 {
-                  v2: [ # message
+                  v3: [ # message
+                    {
+                      UnpaidExecution: {
+                        weightLimit: { 
+                          limited: {
+                            refTime: 5000000000, # 3_000_000_000
+                            proofSize: 1000000, # 1_000_000
+                          },
+                        }
+                      }
+                    },
                     {
                       Transact: {
-                        originType: Superuser,
-                        requireWeightAtMost: 100000000000, # 100_000_000_000
+                        originKind: Superuser,
+                        requireWeightAtMost: {
+                          refTime: 3000000000, # 1_000_000_000 2_440_115_783
```

Comment:
Trying to understand what these comments next to weights mean. As far as I can see we used to just have a more readable version `100000000000` -> `100_000_000_000`, but now there is more info. Some of it seems to be something that is coming up upon test execution, some of it is something else. Would great to have clarity here.

Reply:
my debugging comments, I remove them and leave only readably version